src/sentry/seer/utils.py

```diff
@@ -44,3 +44,43 @@ def detect_breakpoints(breakpoint_request) -> BreakpointResponse:
         headers={"content-type": "application/json;charset=utf-8"},
     )
     return json.loads(response.data)
+
+
+# TODO: change these to NotRequired fields once Python version is 3.11
+class SimilarIssuesEmbeddingsRequestNotRequired(TypedDict, total=False):
+    k: int
+    threshold: int
+
+
+class SimilarIssuesEmbeddingsRequest(SimilarIssuesEmbeddingsRequestNotRequired):
+    group_id: int
+    stacktrace: str
+    message: str
+
+
+class SimilarIssuesEmbeddingsData(TypedDict):
+    parent_group_id: int
+    stacktrace_similarity: float
+    message_similarity: float
+    should_group: bool
+
+
+class SimilarIssuesEmbeddingsReponse(TypedDict):
+    responses: List[SimilarIssuesEmbeddingsData]
+
+
+def get_similar_issues_embeddings(
+    similar_issues_request: SimilarIssuesEmbeddingsRequest,
+) -> SimilarIssuesEmbeddingsReponse | HTTPResponse:
+    """Call /v0/issues/similar-issues endpoint from timeseries-analysis-service."""
+    response = seer_connection_pool.urlopen(
+        "POST",
+        "/v0/issues/similar-issues",
+        body=json.dumps(similar_issues_request),
+        headers={"Content-Type": "application/json;charset=utf-8"},
+    )
+
+    try:
+        return json.loads(response.data.decode("utf-8"))
+    except Exception:
```

Comment:
it is generally not a good idea to except on `Exception`, which will catch everything -- what exact error are you trying to catch?

Reply:
good point, I've changed it to a `AttributeError`